perf/src/bin/perf_client.rs

```diff
@@ -66,6 +66,9 @@ struct Opt {
     /// UDP payload size that the network must be capable of carrying
     #[clap(long, default_value = "1200")]
     initial_max_udp_payload_size: u16,
+    /// Disable packet encryption/decryption (for debugging purpose)
+    #[clap(long = "null-cipher")]
+    nullcipher: bool,
```

Comment:
The field here should follow the naming convention, so null_cipher rather than nullcipher.

Reply:
Fixed (and will be updated later depending on name discussion outcome).